.golangci.yml

```diff
@@ -0,0 +1,27 @@
+run:
+  timeout: "10m"
+linters:
+  disable-all: true
+  enable:
+    # Format
+    - "gofumpt" # Gofumpt checks whether code was gofumpt-ed. Which is a superset of gofmt that handles some cases that gofmt missed.
+    - "gci" # Gci controls Go package import order and makes it always deterministic.
+    - "asciicheck" # Checks that all code identifiers does not have non-ASCII symbols in the name.
+    - "bidichk" # Checks for dangerous unicode character sequences.
+    # Error
+    - "errcheck"
+    # Simplify
+    - "gosimple"
+    - "govet"
+    - "ineffassign"
+    - "staticcheck"
```

Comment:
I would have added testifylint as testify is used in the code

But it could be addressed in another PR, maybe

Reply:
i wanted to do a first batch of rules that could be resolved mostly with automatic tools